argparse/argparse.py

```diff
@@ -140,11 +151,9 @@ def parse_args(self, args=None):
 
     def _parse_args(self, args):
         # add optional args with defaults
-        arg_dest = []
```

Comment:
As far as I can tell, the changes in this file from this point onwards are not needed.  They don't change the behaviour, it's just a different way of accumulating the argument values.  Is that so?